.github/workflows/tests_lkcuda_python.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04]
-        pl_backend: ["lightning_kokkos", "all"]
+        pl_backend: ["lightning_kokkos"]
```

Comment:
Should we be testing all here too, or is it tested elsewhere?

Reply:
Yes, we wanted to keep one instance of `"all"` testing. I would suggest keeping this one and testing only with `pl-device-test` as discussed in Slack.